### DISCUSSION TITLE
How To Delete An Album?

### DISCUSSION THREAD

**Discussion by Andrew-McGee** [ampache/ampache#2992] [Q&A] [answered]
Hi Folks,

Feel a bit stupid but trying to delete a few albums that got imported incorrectly due to wrong tags. All I want to do is remove them so I can re-add them again but struggling to figure out where/how to do that in the GUI or the CLI. Any pointers?

Or do I go straight to the database and delete? I can edit in phphmyadmin but worried I might break something.

Thanks!

**Comment by lachlan-00** [comment, marked as answer]
if it's just tags, update the tags and do a verify. (API has update_from_tags() as well)

If they're in the wrong folder you can delete or move the files and a clean will remove then from the DB.

The songs are static ID's but album numbers will change when the tags change and you do a verify

**Comment by Andrew-McGee** [comment]
What happened was the tracks all had different date tags - it was a compilation album. So when ampache read it in I ended up with heaps of different albums all with the same name but different years. The date was the only thing I could see that differentiated them. It was like each track was in it's own album - even though the albums all had the same name.

I got caught out with verify before. It's my own fault. I made a lot of updates to albums, tracks and artists by editing in the ampache GUI, but I didn't update the tags. Then when I next ran a verify weeks later all my changes got reverted. Lesson learned - sort of. :)

**Reply by lachlan-00** [reply]
That's something that catches a lot of people up. All dates are viewed as an album date so people with mix tapes get caught a lot. but when you view it as an album field the mixtape still has a street date.

Outside of extra code the solution is that year is an album field until it's able to be worked around.
